src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -237,6 +239,22 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
             WorkspaceService.ListAllWorkspaces)
       }
     } ~
+    path("admin" / "publishedWorkspaces") {
```

Comment:
I don't think rawls should have this API. Whoever is calling this should use the admin/workspacesByAttribute api instead.